features/step_definitions/notifications_steps.rb

```diff
@@ -16,3 +16,32 @@
     expect(find(selector, match: :first)).to have_no_css(".create_participation", visible: false)
   end
 end
+
+And "I wait for notifications to load" do
+  expect(find("#notification-dropdown")).to have_no_css(".loading")
+end
+
+And /^I scroll down on the notifications dropdown$/ do
```

Comment:
You changed some of these to normal strings, can you do that everywhere where there is no need for a regex?